odoo/models.py

```diff
@@ -5391,9 +5391,11 @@ def update(self, values):
 
     @api.model
     def flush(self, fnames=None, records=None):
-        """ Process all the pending recomputations (or at least the given field
-            names `fnames` if present) and flush the pending updates to the
-            database.
+        """ When called without parameters, process all the pending computations
+        on all models, and flush the pending updates to the database.
+        If ``fnames`` is given, the processing is limited to the given fields of
+        the current model, and even to the given records (if both ``fnames`` and
+        ``records`` are given).
```

Comment:
It is maybe not the moment/place to discuss docstring guidelines, but I find the PEP257 way easier to read
```python
    def flush(self, fnames=None, records=None):
        """Process the all the pending computations  and flush to the database.
        
        :param fnames (list<string>): list of field names to be flushed. 
            If it is given, the processing is limited to the given fields
            of the current model.
        :param records (Model): the recordset that needs to be processed. 
            It has no effect if ``fnames`` is not given
        """
```